<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="CameraFrameDisplayContainer" name="cameraFrameDisplayWidget" native="true"/>
     </item>
     <item>
      <widget class="MatplotlibWidget" name="gaussianFitPlot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Setup</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="3" rowspan="4">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Gaussian Fit Parameters</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Amplitude:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Centre x:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Rotation:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Centre y</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Width y:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Width x:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="amplitudeLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="centreXLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="centreYLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="widthXLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="widthYLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="rotationLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Mask Offset</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="CameraGainSpinBox" name="gainSpinBox"/>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Gain</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QPushButton" name="acquireMaskButton">
          <property name="text">
           <string>Acquire Mask</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Saturated Pixel Count:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QSpinBox" name="maskOffsetSpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Exposure</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="CameraExposureSpinBox" name="exposureSpinBox"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="saturatedPixelCountLabel">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header>pyjohnstonlab.gui.widgets.matplotlib</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CameraExposureSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>pyjohnstonlab.gui.widgets.camera.cameraexposurespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>CameraFrameDisplayContainer</class>
   <extends>QWidget</extends>
   <header>pyjohnstonlab.gui.widgets.camera.cameraframedisplaycontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CameraGainSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>pyjohnstonlab.gui.widgets.camera.cameragainspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
